src/print.js

```diff
@@ -0,0 +1,3 @@
+export default function printMe() {
+    console.log('I get called from print.js!');
+   }
```

Comment:
- [ ] You have something like this in the browser.
### Your browser
![aaaaa](https://user-images.githubusercontent.com/81727988/194544754-7c6613c4-c785-48f6-b8dc-8f20787d3d8a.png)

Reply:
Ok.

Reply:
This is also resolved. That was the webpack demo added wrongly. I am very sorry for that.

The current output is attached herewith :
https://user-images.githubusercontent.com/106825373/194581781-36efba1c-056b-497d-87b3-b50475cc0b07.png